### ISSUE TITLE
Images Rotating

### ISSUE THREAD

**Issue by GitHubUser5672** [szTheory/exifcleaner#234]
Some images rotate after stripping them of exif data, if you could add an option to disable rotation or something that would be great as this way I have to go through hundreds of files and manually rotate them back to the original :(

**Comment by GitHubUser5672** [comment]
Btw is this project discontinued?

**Comment by szTheory** [comment]
Yes, no plans to update until Electron adds support for ESM node modules

**Comment by ebrannin-bw** [comment]
[Looks like](https://www.electronjs.org/docs/latest/tutorial/esm) ESM was aded in Electron v28.
